.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,192 @@
+name: Bug Report
+description: Broken or unintended behavior with MSAL .NET Authentication Library.
+title: '[Bug] '
+labels: [bug-unconfirmed, question]
+body:
+- type: markdown
+  attributes:
+    value: |
+      ## Before Submitting your Issue
+      Please make sure that your question or issue is not already covered in [MSAL docs](https://learn.microsoft.com/en-us/entra/msal/dotnet/) or [samples](https://learn.microsoft.com/en-us/azure/active-directory/develop/sample-v2-code?tabs=apptype).
+
+      **Logs and network traces**
+      Without logs or traces, it is unlikely that the team can investigate your issue. Capturing logs and network traces is described in [Logging wiki](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/logging).
+
+
+- type: markdown
+  attributes:
+    value: |
+      ## Issue Details   
+    
+- type: input
+  attributes:
+    label: Library Version used
+    description: "Please enter the latest version this issue can be reproduced in."
+    placeholder: "4.54.1"
+  validations:
+    required: true
+    
+- type: dropdown
+  attributes:
+    label: .Net Framework
+    options:
+      - ".NET 4.5 or greater"
+      - ".NET Core 2.1 or greater"
+      - ".NET Standard 2.0"
+      - "Mono Android 9.0"
+      - "UAP 10.0"
+      - "Xamarin.iOS 10"
+      - "Xamarin.Mac 2.0"
+      - "Net6.0-android"
+      - "Net6.0-ios"
+      - "WINUI3"
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: Public, Confidential or Managed Identity Client?
+    description: "Are you using PublicClientApplication,  ConfidentialClientApplication or ManagedIdentityApplication?"
+    multiple: true
+    options: 
+      - "Public"
+      - "Confidential"
+      - "Managed Identity"
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: Application Type?
+    description: "Please select your application type"
+    multiple: true
+    options: 
+      - "Desktop"
+      - "Mobile"
+      - "Web app"
+      - "Daemon app" 
+      - "Other" 
+  validations:
+    required: true   
+
+- type: textarea
```

Comment:
Can you display this only if the user chooses "other" ?

Reply:
unfortunately no I removed it